scripts/bootstrap.sh

```diff
@@ -915,11 +915,12 @@ install_llvm() {
 		install_packages "llvm@$(llvm_version)"
 		;;
 	apk)
+		# alpine doesn't have a lld19 package on 3.21 atm so use bare one for now
 		install_packages \
 			"llvm$(llvm_version)" \
 			"clang$(llvm_version)" \
 			"scudo-malloc" \
-			"lld$(llvm_version)"
+			"lld"
```

Comment:
does it still use lld 19?

Reply:
yeah but alpine 3.21 doesnt offer a `lld19` package atm, only a `lld` package at version 19
we'll have to switch it to 19 when they add 20 the next time we have to rebuild the images

Reply:
its odd because they do have `llvm19` and `clang19` packages but left lld out of the pattern